content/appendices.md

```diff
@@ -663,6 +672,10 @@ Room IDs are case-sensitive. They are not meant to be
 human-readable. They are intended to be treated as fully opaque strings
 by clients.
 
+The localpart of a room ID (`opaque_id` above) may contain any valid
+unicode codepoints except `:`, but it is recommended to only include
```

Comment:
again, we allow ctrl chars?

Reply:
Yeah, probably best to define everything except NUL as allowed like with user IDs